src/NHSDPublicBrowseAcceptanceTestsSpecflow/Steps/BrowseSolutions/ViewASolution.cs

```diff
@@ -176,10 +177,18 @@ public void ThenThePageLastUpdatedDateShownIsUpdatedAsExpected()
         {
             _test.driver.Navigate().Refresh();
             var actualLastUpdated = _test.pages.ViewASolution.GetSolutionLastUpdated();
-            actualLastUpdated = Convert.ToDateTime(actualLastUpdated).ToString("yyyy-MM-dd");
-            actualLastUpdated.Should().Be(expectedLastUpdatedDate);
+
+            // Remove extraneous string from UI last updated
+            var convertedDate = ConvertDateToLongDateTime(actualLastUpdated);
+            
+            convertedDate.Should().Be(expectedLastUpdatedDate);
         }
 
+        private string ConvertDateToLongDateTime(string date)
+        {
+            var reduced = date.Replace("Solution information last updated: ", "");
```

Comment:
take a look in Actions.Pages ViewASolution GetSolutionLastUpdated()

Reply:
I didn't realise we were doing anything there, I'll modify

Reply:
Updated in next commit